tests/test_workers/test_tasks/test_build_merge_index_image.py

```diff
@@ -227,6 +228,7 @@ def test_add_bundles_missing_in_source(
             'csvName': 'bundle5-5.0',
         },
     ]
+    mock_gil.side_effect = ['=v4.5', '=v4.6', 'v4.7', 'v4.5-v4.7', 'v4.5,v4.6']
```

Comment:
Could you add, maybe new test cases, that explicitly check the behavior of `filter_invalid_bundles`?